
# Returns dictionary: transceiver id (key), model name (value)
def get_tx_ids():
    return{
        "1": "glider_1",
        "1": "beacon_1",
        "1": "beacon_2",
        "1": "beacon_3",
        "1": "beacon_4",
        "1": "",
        "1": "",
        "1": "",
        "1": "",
        "1": "",

    }

# Returns dictionary: transponder id (key), model name (value)
def get_rx_ids():
    return{

    }